mod1/mod11/src/file-6.xoo

```diff
@@ -0,0 +1,11 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// violate HasTag rule: xoo on line 11
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:33673/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:33673/project/issues?id=sample&branch=feature4&issues=AWFQs7oRxCAou-Fij-wj&open=AWFQs7oRxCAou-Fij-wj)

[ISSUE_KEY]: AWFQs7oRxCAou-Fij-wj